doc/api/a00242.html

```diff
@@ -89,7 +89,7 @@
 Functions</h2></td></tr>
 <tr class="memitem:ga071566cadc7505455e611f2a0353f4d4"><td class="memTemplParams" colspan="2">template&lt;length_t L, typename T , qualifier Q&gt; </td></tr>
 <tr class="memitem:ga071566cadc7505455e611f2a0353f4d4"><td class="memTemplItemLeft" align="right" valign="top">GLM_FUNC_DECL vec&lt; L, T, Q &gt;&#160;</td><td class="memTemplItemRight" valign="bottom"><a class="el" href="a00242.html#ga071566cadc7505455e611f2a0353f4d4">exp</a> (vec&lt; L, T, Q &gt; const &amp;v)</td></tr>
-<tr class="memdesc:ga071566cadc7505455e611f2a0353f4d4"><td class="mdescLeft">&#160;</td><td class="mdescRight">Returns the natural exponentiation of x, i.e., e^x.  <a href="a00242.html#ga071566cadc7505455e611f2a0353f4d4">More...</a><br /></td></tr>
+<tr class="memdesc:ga071566cadc7505455e611f2a0353f4d4"><td class="mdescLeft">&#160;</td><td class="mdescRight">Returns the natural exponentiation of v, i.e., e^v.  <a href="a00242.html#ga071566cadc7505455e611f2a0353f4d4">More...</a><br /></td></tr>
```

Comment:
The content of the `doc/api/` directory is generated by doxygen, you should fix the comment in the actual source, and then regenerate the documentation. The typo is here:

https://github.com/g-truc/glm/blob/cc98465e3508535ba8c7f6208df934c156a018dc/glm/exponential.hpp#L38

Reply:
Thanks for pointing that out. I'll change it under glm/glm/exponential.hpp as you pointed out.